src/js/setUpSite.js

```diff
@@ -138,7 +124,8 @@ const loadParkingLot = async (cityId, parkingLayer) => {
     .getLayers()
     .find((city) => city.feature.properties.id === cityId);
   if (!alreadyLoaded) {
-    parkingLayer.addData(await parkingLots[`${cityId}.geojson`]());
+    const lots = await parkingLots;
+    parkingLayer.addData(await lots[`${cityId}.geojson`]());
```

Comment:
Why this change? Btw the dynamic import is so that we lazily load the GeoJSON file because they are large and loading everything eagerly slows down the site too much. Implemented recently in https://github.com/ParkingReformNetwork/parking-lot-map/pull/142.

Reply:
I'm reverting it.  I thought it solved an issue I was having with an error I was getting in the console but I can't seem to reproduce the error.  I'm finding it a bit difficult to make sure the version of the app I have served is up to date with my changes and I've gotten what I think is a corrupted cache a number of times (weird segfauls/mutex lock complaints?).  Anyway sorry that crept in.

Reply:
Hm, possibly Parcel bugs. Sometimes it can help to `rm -rf .parcel-cache`.

Reply:
Yeah that's what I ended up doing.